.travis.yml

```diff
@@ -1,28 +1,13 @@
-sudo: false
 language: ruby
 rvm:
-  - 1.9
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.3.1
+  - 2.2.7
+  - 2.3.5
+  - 2.4.2
+  - 2.5.0
```

Comment:
What do you think about adding `ruby-head`?

Reply:
Only if you're adding Rails.master. You're testing a dependency of a dependency at that point. Plus you need to allow failures of it too. Not much of a value.

Reply:
I don't like getting blindsided by bugs whenever a new version of Ruby is released (even a prominent project like spring [is susceptible](https://github.com/rails/spring/issues/550), it seems), and therefore would appreciate the advance notice that `ruby-head` would give us. And indeed, Rails `5-1-stable` does in fact test against `ruby-head`. But it is an allowed failure. If we that means we have to make it an allowed failure as well, than I agree it is of little value. I'll leave it up to you. Thanks for the feedback!

Reply:
Easy enough to add ruby-head. Travis will run a bit slower. I'll add one bleeding-edge job.